packages/quantic/force-app/main/default/lwc/quanticUtils/quanticUtils.js

```diff
@@ -562,8 +562,12 @@ export function getLastFocusableElement(element) {
     return null;
   }
 
+  if (element.tagName === "SLOT" && /** @type {HTMLSlotElement} */(element).assignedElements().length) {
+    return getLastFocusableElementFromSlot(/** @type {HTMLSlotElement} */(element))
+  }
+
   const focusableElements = Array.from(element.childNodes)
```

Comment:
Wouldn't it be possible to just mark the `focusableElements` as `@type {HTMLElement[]}` instead?

Reply:
Actually the problem here is not the type of the array `focusableElements`. the problem is in the return value of `element.childNodes` which returns an array of type `ChildNodes[]`.
The values of `element.childNodes` are then passed to the function `getLastFocusableElement` which only accepts the type `HTMLElement`. 
When I try to mark `element.childNodes` as `@type {HTMLElement[]}`, I get the following error: `Type 'ChildNode[]' is not assignable to type 'HTMLElement[]'`
That's why I'm using this inline type annotation, its the only way around that I found.